pkg/volume/cinder/attacher.go

```diff
@@ -124,7 +124,18 @@ func (attacher *cinderDiskAttacher) VolumesAreAttached(specs []*volume.Spec, nod
 		volumesAttachedCheck[spec] = true
 		volumeSpecMap[volumeSource.VolumeID] = spec
 	}
-	attachedResult, err := attacher.cinderProvider.DisksAreAttached(volumeIDList, string(nodeName))
+	instances, res := attacher.cinderProvider.Instances()
+	if !res {
+		return volumesAttachedCheck, fmt.Errorf("underlying cloud provider does not claim to support Instances api")
+	}
+	instanceid, err := instances.InstanceID(nodeName)
+	if err != nil {
+		return volumesAttachedCheck, err
+	}
+	if ind := strings.LastIndex(instanceid, "/"); ind >= 0 {
```

Comment:
How is `nodeName` different to `instanceId`? Why would `instanceId` contain slash?

@k8s-sig-openstack-pr-reviews, PTAL.